release/models/bgp/openconfig-bgp-common-multiprotocol.yang

```diff
@@ -24,7 +24,22 @@ submodule openconfig-bgp-common-multiprotocol {
     for multiple protocols in BGP. The groupings are common across
     multiple contexts.";
 
-  oc-ext:openconfig-version "9.2.0";
+  oc-ext:openconfig-version "10.0.0";
```

Comment:
```suggestion
  oc-ext:openconfig-version "9.4.0";
```

Because this is a non breaking change, only the minor version needs to be incremented